src/sitemap.njk

```diff
@@ -0,0 +1,19 @@
+---
+permalink: /sitemap.xml
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    {% for page in collections.all %}
+    {%- if page.url and page.data.locale == "en-CA" -%}
```

Comment:
Page.data.locale is undefined in all cases, so sitemap isn't generated properly. Removing the second check on locale fixes the issue it seems.